cocotb/share/lib/vpi/VpiImpl.cpp

```diff
@@ -296,13 +297,52 @@ GpiObjHdl* VpiImpl::native_check_create(void *raw_hdl, GpiObjHdl *parent)
 GpiObjHdl* VpiImpl::native_check_create(std::string &name, GpiObjHdl *parent)
 {
     vpiHandle new_hdl;
+    const vpiHandle parent_hdl = parent->get_handle<vpiHandle>();
```

Comment:
Should we protect against casting a non-VPI parent handle to `vpiHandle`?
I'm not sure how a `vpiGenScopeArray` could have a non-VPI parent.

Reply:
I don't think we need to worry about that in this PR. It's systemic.